testing/tests/ui/let_destructuring_has_rest.rs

```diff
@@ -0,0 +1,40 @@
+use rinja::Template;
+
+struct X {
+    a: u32,
+    b: u32,
+}
+
+#[derive(Template)]
+#[template(source = "
+{%- if let X { a, .. } = x -%}hello {{ a }}{%- endif -%}
+", ext = "html")]
+struct T1 {
+    x: X,
+}
+
+#[derive(Template)]
+#[template(source = "
+{%- if let X { a, .., } = x -%}hello {{ a }}{%- endif -%}
+", ext = "html")]
+struct T2 {
+    x: X,
+}
+
+#[derive(Template)]
+#[template(source = "
+{%- if let X { a .. } = x -%}hello {{ a }}{%- endif -%}
+", ext = "html")]
+struct T3 {
+    x: X,
+}
+
+#[derive(Template)]
+#[template(source = "
+{%- if let X { .. } = x -%}hello {{ a }}{%- endif -%}
```

Comment:
This code is valid until the `{{ a }}` part. Maybe fix it and move it in non-ui tests?

Reply:
Yep, moved.